README.rst

```diff
@@ -40,6 +40,14 @@ Usage 📖
 
 .. image:: https://github.com/JaDogg/pydoro/raw/master/images/animation.gif
 
+Configuration
```

Comment:
Remove this for now. 

I like your idea of maintaining set of configs internally.
Your data model is very good too. 

However I prefer to use a config file such as `~\.pydoro.ini`. That way end user doesn't need to edit code to change keybindings. And we can also introduce any new features such as different time periods to same ini file as well.